README.md

```diff
@@ -1,3 +1,6 @@
+7/14 MOOOOORE changes
```

Comment:
Too many O

Reply:
Thanks for your feedback. I'll change it and push the new file onto my branch

Reply:
DONE